data/changelog.txt

```diff
@@ -495,6 +513,7 @@ Adjust MRE caloric value to reflect real MREs.
 Lower max stat cap in character creation to 14.
 Overhaul archery balance.
```

Comment:
Archery overhaul completely breaks archery craftable progression and makes you rely on getting extraordinarily lucky with world spawns to progress to lategame content

Reply:
Gotta find a hunting store Insane :^)